sword-server/src/main/java/edu/unc/lib/dl/cdr/sword/server/managers/ServiceDocumentManagerImpl.java

```diff
@@ -113,39 +124,39 @@ public ServiceDocument getServiceDocument(String sdUri, AuthCredentials auth, Sw
      */
     protected List<SwordCollection> getImmediateContainerChildren(PID pid, AuthCredentials auth,
             SwordConfigurationImpl config) throws IOException {
```

Comment:
Are we still using SWORD for anything?

Reply:
Yes, forms deposits, maybe some other types of deposits too, but i'd need to check